File: JavaScript/cryptowarp_getdata.js

//  The code below is an example of how to get data from the cryptowarp.io platform API.

const axios = require('axios');

// the symbol
const symbol = "BTCUSDT";

// the api key
const api_key = "YOUR_API_KEY";

// the headers
const headers = {
    "Authorization": "Bearer " + api_key
};

// the url
const url = "https://cryptowarp.io/api/models/" + symbol;

// get the data
const response = axios.get(url, {
    headers: headers
});

// handle the response
response.then(function (response) {
    console.log(response.data);
}
).catch(function (error) {
    console.log(error.response.data);
});

File: Elixir/cryptowarp_getdata/lib/cryptowarp_getdata.ex
defmodule CryptowarpGetdata do
  # The code below is an example of how to get data from the cryptowarp.io platform API.

  def get_data() do
    # the simbol
    symbol = "BTCUSDT"

    # the url
    url = "https://cryptowarp.io/api/models/" <> symbol

    # the api key
    api_key = "YOUR_API_KEY"

    # the headers
    headers = [{"Authorization", "Bearer #{api_key}"}, {"Content-Type", "application/json"}]

    case HTTPoison.get!(url, headers) do
      %HTTPoison.Response{status_code: 200, headers: _resp_headers, body: body} ->
        Poison.decode!(body)

      %HTTPoison.Response{status_code: _, body: body} ->
        {:error, body}
    end
  end
end
